src/main/diagnostics/diagnostics.ts

```diff
@@ -0,0 +1,48 @@
+import yaml from 'yaml';
```

Comment:
This import appears to be unused. That would actually be caught by lint now (pending rebase) :)

Reply:
Yes, for some reason I forgot how to use yaml in node.js.

Reply:
Because the test calls `yaml.parse()` for your fake data :)